internal/vds/vds.go

```diff
@@ -96,6 +96,10 @@ type Metadata struct {
 
 // @Description Fence metadata
 type FenceMetadata struct {
+	// Data format is represented by a numpy-style formatcodes. E.g. f4 is 4
+	// byte float, u1 is 1 byte unsinged int and u2 is 2 byte usigned int
```

Comment:
Just note that according to our calculations, this is always float, even if original data is u2/u1.